Modules/Effect/Bars/Bars.cs

```diff
@@ -2573,8 +2572,11 @@ private void GetRotatedPosition(ref int x, ref int y, double angle, bool swapXY)
 	        // Transform (rotate) the point
 	        System.Windows.Point transformedPoint = rt.Transform(tempPoint);
 
-	        // Updated the x and y coordinates for the rotation
-	        x = (int)Math.Round(transformedPoint.X);
+            //transformedPoint.X -= xOffset;
+            //transformedPoint.Y -= yOffset;
+            

```

Comment:
Looks like the previous commit removed these two commented lines, and then they are put back here. Was the intent to remove them?